CONTRIBUTING.md

```diff
@@ -1,3 +1,5 @@
+<a href="https://www.phorm.ai/query?projectId=bebfbe39-2f45-4d49-88fd-3de89c49c301"><img src="https://stuff.charm.sh/misc/phorm-badge.svg" alt="phorm.ai"></a>
```

Comment:
Put the badge below the comment, the comment should be on top

Reply:
I've moved it below the comment.